src/lib/factories/leads.ts

```diff
@@ -64,3 +86,10 @@ export const SearchCallLead = (attributes = {}): SearchCallLeadType => ({
   ...searchCallLeadDefaults,
   ...attributes,
 })
+
+export const SearchWhatsAppLead = (
+  attributes = {}
```

Comment:
I know you followed the pattern of the other factories but we could define `attributes` from being of the type `Partial<SearchWhatsAppLeadType>` with that we can support the consumer from passing the correct attributes. What do you think?